### ISSUE TITLE
Respect XDG directory specification for $HOME/.stremio-server

### ISSUE THREAD

**Issue by otuva** [Stremio/stremio-bugs#448]
**Describe the bug**
$HOME/.stremio-server/ is used to store application data. It would be great if XDG standards are respected. This would help to:

- declutter the $HOME 
- create more structured backups since the user can differentiate between cache, config and user data

**Expected behavior**
Storing:
- config -> $XDG_CONFIG_HOME | (default is $HOME/.config)
- cache -> $XDG_CACHE_HOME | (default is $HOME/.cache )
- user data -> $XDG_DATA_HOME | (default is $HOME/.local/share) 
- and creating a fallback to $HOME if XDG directories are not present

**Additional context**

- [Freedesktop xdg specs](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html)
- [Related arch wiki page](https://wiki.archlinux.org/title/XDG_Base_Directory)

**Comment by TRtomasz** [comment]
wouldn't consider it a bug, it would be a new feature

**Comment by otuva** [comment]
Yeah, you're probably right. Is this viable, though?

**Comment by TRtomasz** [comment]
it is possible but would require changes in few components, Stremio shell and streaming server

**Comment by Gitleptune** [comment]
I agree, this would be awesome to add as it cluttering the home directory is really annoying.

**Comment by TRtomasz** [comment]
Closing this issue as we have released new app version https://blog.stremio.com/stremio-5-released-for-linux/
